lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -234,28 +234,33 @@ defmodule ExUnit.Callbacks do
   end
 
   @doc false
-  def __merge__(_mod, context, :ok) do
+  def __merge__(mod, context, value) do
+    __merge__(mod, context, value, value)
+  end
+
+  @doc false
+  def __merge__(_mod, context, :ok, _original_value) do
```

Comment:
Hey Andrea :wave:. Yes, it gets called on on `__MODULE__`from [`compile_merge/2`](https://github.com/jeffkreeftmeijer/elixir/blob/188a3e9e51c11a7680de2e1c1c1469bfd54b7c2d/lib/ex_unit/lib/ex_unit/callbacks.ex#L315):

``` elixir
defp compile_merge({callback, nil}) do
  quote do
    unquote(__MODULE__).__merge__(__MODULE__, context, unquote(callback)(context))
  end
end
```